kodilla-stream/src/test/java/com/kodilla/exception/homework/WarehouseTestSuite.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.exception.homework;
+
+import org.junit.Test;
+
+public class WarehouseTestSuite {
+
+    @Test(expected = OrderDoesntExistException.class)
+    public void testIfOrderExist_withException() throws OrderDoesntExistException {
+        //given
+        Warehouse warehouse = new Warehouse();
+        //when
+        Order ifOrderExist = Warehouse.getOrder("000ORDER");
```

Comment:
to przypisanie `Order ifOrderExist = ` jest niepotrzebne
i tak ifOrderExist nie zostanie nigdy wykorzystany